Client/Card.ts

```diff
@@ -10,6 +10,9 @@ export class Card extends rest.Collection<gracely.Error> {
 	async getImage(token: modelCard.Token): Promise<string | gracely.Error> {
 		return await this.client.get<string>(`card/${token}`, { accept: ["image/svg+xml"] })
 	}
+	async getPDF(token: modelCard.Token): Promise<string | gracely.Error> {
```

Comment:
For abbrivation longer than 2 lettters only capitilize the first letter. Should be `getPdf`.